storybook/src/stories/UseResizeObserver/TextArea.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from 'react'
+import { useResizeObserver } from 'too-many-hooks'
+
+interface Props {
+  onResize: (name?: string | null) => string
+}
+
+const TextArea: React.FC<Props> = ({ onResize }) => {
```

Comment:
Any reason we need both of these stories? We could just combine them by adding the resize actions to the flex direction story.

Reply:
I think it makes sense to keep both since a `textarea` is basically the only element that has resize enabled by default so it is probably the most common thing someone would want to watch resize events on. Then the container query shows off other things that a resize observer is good for. One for searchability/relatability and one for potential.